client/src/views/CodeView.vue

```diff
@@ -97,6 +97,16 @@ export default {
       monaco.editor.setModelLanguage(models[0], this.language);
       models[0].setValue(this.default_code[this.language]);
       this.status = "// Status";
+
+      // Disable minimap and overview ruler if language is SQL
+      const editors = monaco.editor.getEditors();
+      if (this.language === "sql") {
```

Comment:
nit: Regarding "sql", should we define constants somewhere and use it here.

Reply:
Is a `.vue` file Javascript or Typescript?  If it's Javascript then these sorts of constants are really needed (since there is no compilation step there is no real advantage to using them).  If it's Typescript then I agree that having this kind of constant would be a good idea.

Reply:
Right now, the `.vue` files are in JavaScript, the project was initiated on JavaScript but can be modified to be in TypeScript.